### ISSUE TITLE
Conda cripples itself on update

### ISSUE THREAD

**Issue by MikeDacre** [conda/conda#5454]
I have anaconda installed to /usr/local/anaconda3, I ran:

    conda update conda anaconda pandas numpy scipy statsmodels matplotlib

I thought everything worked fine, but then jupyterhub was behaving strangely, so I restarted it.. except it failed:

```
● jupyterhub.service - Jupyter Hub
   Loaded: loaded (/usr/local/anaconda3/etc/jupyter/../../../jupyter/jupyterhub.service; bad; vendor preset: disabled)
   Active: failed (Result: exit-code) since Tue 2017-05-30 10:20:57 PDT; 33s ago
  Process: 31918 ExecStart=/usr/local/anaconda3/bin/jupyterhub -f /usr/local/anaconda3/etc/jupyter/jupyterhub_config.py (code=exited, status=1/FAILURE)
 Main PID: 31918 (code=exited, status=1/FAILURE)

May 30 10:20:57 fraser-server systemd[1]: Started Jupyter Hub.
May 30 10:20:57 fraser-server jupyterhub[31918]: Traceback (most recent call last):
May 30 10:20:57 fraser-server jupyterhub[31918]:   File "/usr/local/anaconda3/bin/jupyterhub", line 3, in <module>
May 30 10:20:57 fraser-server jupyterhub[31918]:     from jupyterhub.app import main
May 30 10:20:57 fraser-server jupyterhub[31918]: ImportError: No module named 'jupyterhub'
May 30 10:20:57 fraser-server systemd[1]: jupyterhub.service: Main process exited, code=exited, status=1/FAILURE
May 30 10:20:57 fraser-server systemd[1]: jupyterhub.service: Unit entered failed state.
May 30 10:20:57 fraser-server systemd[1]: jupyterhub.service: Failed with result 'exit-code'.
```

So I tried to `conda search jupyterhub` and got this:

```
Traceback (most recent call last):
  File "/usr/local/anaconda3/bin/conda", line 4, in <module>
    import conda.cli
ImportError: No module named 'conda'
```

I confirmed that my PATH definitely set properly and the python is `/usr/local/anaconda3/bin/python`,  so I fired up python, but the welcome message was:

```
Python 3.5.2 (default, Nov  7 2016, 11:31:36)
[GCC 6.2.1 20160830] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

Which doesn't look like an anaconda python. I looked around in the `anaconda/bin` directory and everything looked fine.

My guess is that this is related to #2463 so I tried the 'fix' linked in that comment: https://github.com/conda/conda/issues/1733#issuecomment-150641779, basically reinstalling anaconda and then rsyncing in all the changes. That failed, as it overwrote the fixed files with the broken ones.

So at this point it looks like updating a few packages with conda destroyed the whole install and I am going to have to start from scratch, which is a little frustrating to say the least.

**Comment by MikeDacre** [comment]
I ended up fixing this by reinstalling anaconda, installing and updating all the packages I needed, and the rsycing just the etc/ directory with the config files. That got me back most of the way to where I was before.

Still though, does anyone know why this would happen?

**Comment by kalefranz** [comment]
Do you know what version of conda you were using?  Conda uninstalling itself should be fixed in all cases for conda >= 4.3.

**Comment by MikeDacre** [comment]
Thanks for the answer @kalefranz. I was using 4.2.0 when I installed anaconda originally a few months ago, I had run conda update conda a few times, but not for at least two months, so I am not sure exactly which version I was using. I am just going to assume it was < 4.3 though and close this.
